x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart_annotations.ts

```diff
@@ -132,6 +132,19 @@ export function renderAnnotations(
   const levelHeight = ANNOTATION_LEVEL_HEIGHT;
   const levels = getAnnotationLevels(focusAnnotationData);
 
+  const onAnnotationMouseOver = function (this: object, d: Annotation) {
+    showFocusChartTooltip(d, this);
+  };
+
+  const onAnnotationClick = (d: Annotation) => {
+    // clear a possible existing annotation set up for editing before setting the new one.
```

Comment:
```suggestion
    // clear a possible existing annotation setup for editing before setting the new one.
```

Reply:
Re-reading this again and it does seems like `set up` is probably correct here. Looks like there was any existing annotation previously typed in (but not applied) in the flyout, we need to clear it out.